spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,63 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validation" do
+    it { should validate_presence_of(:title) }
+    it { should validate_length_of(:title).
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.